[BUG] npm包weapp-navigation-bar在较新版本的remax上无法正常渲染 (应该不是这个包本身的问题)

**bug 描述**
在1.4.1版本中可正常工作, 升级到1.5.0后失效, 不正常渲染, 但slot的内容渲染了

**复现步骤**
为方便复现, 可以直接使用复现代码中提供的仓库地址.
一共2个commit, 1.4.1的, 然后是升级到1.5.0的.
npm run dev, 可以看到组件并没有渲染.
reset到第一个commit, 然后dev, 即可看到正常渲染的组件.

**期望结果**
组件正常渲染

**复现代码**
[https://github.com/WhiteMinds/remax-bug-demo](https://github.com/WhiteMinds/remax-bug-demo)

**版本信息:**

- remax 版本: [1.5.0]
- 手机型号 [iPhone X]
- 小程序端 [微信小程序]
- 小程序版本[微信开发者工具]
- 开发环境 [mac OS]

**其他信息**
1.4.1:
![1.4.1](https://user-images.githubusercontent.com/9160743/69206408-8484af00-0b87-11ea-9c1a-1c68c781708b.png)
1.5.0:
![1.5.0](https://user-images.githubusercontent.com/9160743/69206416-88b0cc80-0b87-11ea-8e78-ec26b130d2b8.png)

> @WhiteMinds 试试把这个 `e` 打印出来看看
> 
> https://github.com/remaxjs/remax/blob/master/packages/remax-cli/src/build/index.ts#L26

```
/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/macro.js:1
Error: Cannot find module 'babel-plugin-macros'
Require stack:
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/macro.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/index.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax/cjs/propsAlias.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/build/adapters/wechat/index.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/build/index.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/index.js
- /Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/bin/remax
    at Object.<anonymous> (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/macro.js:10:29)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/macro.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax.macro/lib/index.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax/cjs/propsAlias.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/build/adapters/wechat/index.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/build/index.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/index.js',
    '/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/bin/remax'
  ]
}
Error: Target wechat is not supported yet.
    at Object.exports.default (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/build/index.js:33:15)
    at Object.handler (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/index.js:14:71)
    at Object.runCommand (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/yargs/lib/command.js:242:26)
    at Object.parseArgs [as _parseArgs] (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/yargs/yargs.js:1078:30)
    at Object.parse (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/yargs/yargs.js:566:25)
    at Object.run (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/lib/index.js:29:10)
    at Object.<anonymous> (/Users/whitemind/git-repos/nutstore-assignment-assist/mini-program/node_modules/remax-cli/bin/remax:3:28)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
error Command failed with exit code 1.
```

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/app.json
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/app.json
@@ -2,7 +2,8 @@
   "pages": [
     "pages/index",
     "pages/index2",
-    "pages/index3"
+    "pages/index3",
+    "pages/index4"
   ],
   "plugins": {
     "myPlugin": {

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.axml
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.axml
@@ -1,0 +1,850 @@
+<import-sjs name="helper" from="../helper.sjs" />
+<template is="REMAX_TPL" data="{{tree: helper.reduce(action)}}" />
+
+<template name="REMAX_TPL">
+  <block a:for="{{tree.root.children}}" a:key="{{item.id}}">
+    <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+  </block>
+</template>
+
+
+  <template name="REMAX_TPL_1_a-0">
+  <a-0  foo="{{item.props['foo']}}"  title="{{item.props['title']}}" ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </a-0>
+</template>
+  
+  <template name="REMAX_TPL_1_a-1">
+  <a-1 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </a-1>
+</template>
+  
+  <template name="REMAX_TPL_1_b-0">
+  <b-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </b-0>
+</template>
+  
+  <template name="REMAX_TPL_1_button">
+  <button  class="{{item.props['class']}}"  size="{{item.props['size']}}"  type="{{item.props['type']}}"  plain="{{item.props['plain']}}"  disabled="{{item.props['disabled']}}"  loading="{{item.props['loading']}}"  hover-class="{{item.props['hover-class']}}"  hover-start-time="{{item.props['hover-start-time']}}"  hover-stay-time="{{item.props['hover-stay-time']}}"  hover-stop-propagation="{{item.props['hover-stop-propagation']}}"  form-type="{{item.props['form-type']}}"  open-type="{{item.props['open-type']}}"  scope="{{item.props['scope']}}"  onTap="{{item.props['onTap']}}"  app-parameter="{{item.props['app-parameter']}}"  public-id="{{item.props['public-id']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </button>
+</template>
+  
+  <template name="REMAX_TPL_1_c-0">
+  <c-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </c-0>
+</template>
+  
+  <template name="REMAX_TPL_1_canvas">
+  <canvas  id="{{item.props['id']}}"  style="{{item.props['style']}}"  class="{{item.props['class']}}"  width="{{item.props['width']}}"  height="{{item.props['height']}}"  disable-scroll="{{item.props['disable-scroll']}}"  onTap="{{item.props['onTap']}}"  onTouchStart="{{item.props['onTouchStart']}}"  onTouchMove="{{item.props['onTouchMove']}}"  onTouchEnd="{{item.props['onTouchEnd']}}"  onTouchCancel="{{item.props['onTouchCancel']}}"  onLongTap="{{item.props['onLongTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </canvas>
+</template>
+  
+  <template name="REMAX_TPL_1_checkbox">
+  <checkbox  class="{{item.props['class']}}"  style="{{item.props['style']}}"  value="{{item.props['value']}}"  checked="{{item.props['checked']}}"  disabled="{{item.props['disabled']}}"  onChange="{{item.props['onChange']}}"  color="{{item.props['color']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </checkbox>
+</template>
+  
+  <template name="REMAX_TPL_1_checkbox-group">
+  <checkbox-group  class="{{item.props['class']}}"  style="{{item.props['style']}}"  name="{{item.props['name']}}"  onChange="{{item.props['onChange']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </checkbox-group>
+</template>
+  
+  <template name="REMAX_TPL_1_complex-0">
+  <complex-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </complex-0>
+</template>
+  
+  <template name="REMAX_TPL_1_contact-button">
+  <contact-button  class="{{item.props['class']}}"  style="{{item.props['style']}}"  tnt-inst-id="{{item.props['tnt-inst-id']}}"  scene="{{item.props['scene']}}"  size="{{item.props['size']}}"  color="{{item.props['color']}}"  icon="{{item.props['icon']}}"  alipay-card-no="{{item.props['alipay-card-no']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </contact-button>
+</template>
+  
+  <template name="REMAX_TPL_1_cover-image">
+  <cover-image  class="{{item.props['class']}}"  src="{{item.props['src']}}"  style="{{item.props['style']}}"  onTap="{{item.props['onTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </cover-image>
+</template>
+  
+  <template name="REMAX_TPL_1_cover-view">
+  <cover-view  class="{{item.props['class']}}"  style="{{item.props['style']}}"  onTap="{{item.props['onTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </cover-view>
+</template>
+  
+  <template name="REMAX_TPL_1_d-0">
+  <d-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </d-0>
+</template>
+  
+  <template name="REMAX_TPL_1_e-0">
+  <e-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </e-0>
+</template>
+  
+  <template name="REMAX_TPL_1_f-0">
+  <f-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </f-0>
+</template>
+  
+  <template name="REMAX_TPL_1_form">
+  <form  class="{{item.props['class']}}"  style="{{item.props['style']}}"  report-submit="{{item.props['report-submit']}}"  onSubmit="{{item.props['onSubmit']}}"  onReset="{{item.props['onReset']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </form>
+</template>
+  
+  <template name="REMAX_TPL_1_icon">
+  <icon  class="{{item.props['class']}}"  style="{{item.props['style']}}"  type="{{item.props['type']}}"  size="{{item.props['size']}}"  color="{{item.props['color']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </icon>
+</template>
+  
+  <template name="REMAX_TPL_1_image">
+  <image  src="{{item.props['src']}}"  mode="{{item.props['mode']}}"  class="{{item.props['class']}}"  style="{{item.props['style']}}"  lazy-load="{{item.props['lazy-load']}}"  onLoad="{{item.props['onLoad']}}"  onError="{{item.props['onError']}}"  onTap="{{item.props['onTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </image>
+</template>
+  
+  <template name="REMAX_TPL_1_input">
+  <input  class="{{item.props['class']}}"  style="{{item.props['style']}}"  value="{{item.props['value']}}"  name="{{item.props['name']}}"  type="{{item.props['type']}}"  password="{{item.props['password']}}"  placeholder="{{item.props['placeholder']}}"  placeholder-style="{{item.props['placeholder-style']}}"  placeholder-class="{{item.props['placeholder-class']}}"  disabled="{{item.props['disabled']}}"  maxlength="{{item.props['maxlength']}}"  focus="{{item.props['focus']}}"  confirm-type="{{item.props['confirm-type']}}"  confirm-hold="{{item.props['confirm-hold']}}"  cursor="{{item.props['cursor']}}"  selection-start="{{item.props['selection-start']}}"  selection-end="{{item.props['selection-end']}}"  random-number="{{item.props['random-number']}}"  controlled="{{item.props['controlled']}}"  onInput="{{item.props['onInput']}}"  onConfirm="{{item.props['onConfirm']}}"  onFocus="{{item.props['onFocus']}}"  onBlur="{{item.props['onBlur']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </input>
+</template>
+  
+  <template name="REMAX_TPL_1_label">
+  <label  class="{{item.props['class']}}"  style="{{item.props['style']}}"  for="{{item.props['for']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </label>
+</template>
+  
+  <template name="REMAX_TPL_1_lifestyle">
+  <lifestyle  public-id="{{item.props['public-id']}}"  onFollow="{{item.props['onFollow']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </lifestyle>
+</template>
+  
+  <template name="REMAX_TPL_1_map">
+  <map  style="{{item.props['style']}}"  class="{{item.props['class']}}"  latitude="{{item.props['latitude']}}"  longitude="{{item.props['longitude']}}"  scale="{{item.props['scale']}}"  markers="{{item.props['markers']}}"  polyline="{{item.props['polyline']}}"  circles="{{item.props['circles']}}"  controls="{{item.props['controls']}}"  polygon="{{item.props['polygon']}}"  show-location="{{item.props['show-location']}}"  include-points="{{item.props['include-points']}}"  include-padding="{{item.props['include-padding']}}"  ground-overlays="{{item.props['ground-overlays']}}"  tile-overlay="{{item.props['tile-overlay']}}"  setting="{{item.props['setting']}}"  onMarkerTap="{{item.props['onMarkerTap']}}"  onCalloutTap="{{item.props['onCalloutTap']}}"  onControlTap="{{item.props['onControlTap']}}"  onRegionChange="{{item.props['onRegionChange']}}"  onTap="{{item.props['onTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </map>
+</template>
+  
+  <template name="REMAX_TPL_1_movable-area">
+  <movable-area  class="{{item.props['class']}}"  style="{{item.props['style']}}"  width="{{item.props['width']}}"  height="{{item.props['height']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </movable-area>
+</template>
+  
+  <template name="REMAX_TPL_1_movable-view">
+  <movable-view  class="{{item.props['class']}}"  style="{{item.props['style']}}"  direction="{{item.props['direction']}}"  width="{{item.props['width']}}"  height="{{item.props['height']}}"  x="{{item.props['x']}}"  y="{{item.props['y']}}"  disabled="{{item.props['disabled']}}"  onTouchStart="{{item.props['onTouchStart']}}"  onTouchMove="{{item.props['onTouchMove']}}"  onTouchEnd="{{item.props['onTouchEnd']}}"  onTouchCancel="{{item.props['onTouchCancel']}}"  onChange="{{item.props['onChange']}}"  onChangeEnd="{{item.props['onChangeEnd']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </movable-view>
+</template>
+  
+  <template name="REMAX_TPL_1_my-plugin-0">
+  <my-plugin-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </my-plugin-0>
+</template>
+  
+  <template name="REMAX_TPL_1_my-plugin-1">
+  <my-plugin-1 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </my-plugin-1>
+</template>
+  
+  <template name="REMAX_TPL_1_navigator">
+  <navigator  class="{{item.props['class']}}"  style="{{item.props['style']}}"  open-type="{{item.props['open-type']}}"  hover-class="{{item.props['hover-class']}}"  hover-start-time="{{item.props['hover-start-time']}}"  hover-stay-time="{{item.props['hover-stay-time']}}"  url="{{item.props['url']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </navigator>
+</template>
+  
+  <template name="REMAX_TPL_1_picker">
+  <picker  class="{{item.props['class']}}"  style="{{item.props['style']}}"  range="{{item.props['range']}}"  range-key="{{item.props['range-key']}}"  value="{{item.props['value']}}"  onChange="{{item.props['onChange']}}"  disabled="{{item.props['disabled']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <view>
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item.children[0]}}" />
+    </view>
+    
+  </picker>
+</template>
+  
+  <template name="REMAX_TPL_1_picker-view">
+  <picker-view  class="{{item.props['class']}}"  style="{{item.props['style']}}"  value="{{item.props['value']}}"  indicator-style="{{item.props['indicator-style']}}"  indicator-class="{{item.props['indicator-class']}}"  mask-style="{{item.props['mask-style']}}"  mask-class="{{item.props['mask-class']}}"  onChange="{{item.props['onChange']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <picker-view-column>
+        <view a:for="{{item.children}}">
+          <template is="REMAX_TPL_1_CONTAINER" data="{{item}}" />
+        </view>
+      </picker-view-column>
+    </block>
+    
+  </picker-view>
+</template>
+  
+  <template name="REMAX_TPL_1_progress">
+  <progress  class="{{item.props['class']}}"  style="{{item.props['style']}}"  percent="{{item.props['percent']}}"  show-info="{{item.props['show-info']}}"  stroke-width="{{item.props['stroke-width']}}"  active-color="{{item.props['active-color']}}"  background-color="{{item.props['background-color']}}"  active="{{item.props['active']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </progress>
+</template>
+  
+  <template name="REMAX_TPL_1_radio">
+  <radio  class="{{item.props['class']}}"  style="{{item.props['style']}}"  value="{{item.props['value']}}"  checked="{{item.props['checked']}}"  disabled="{{item.props['disabled']}}"  color="{{item.props['color']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </radio>
+</template>
+  
+  <template name="REMAX_TPL_1_radio-group">
+  <radio-group  class="{{item.props['class']}}"  style="{{item.props['style']}}"  onChange="{{item.props['onChange']}}"  name="{{item.props['name']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </radio-group>
+</template>
+  
+  <template name="REMAX_TPL_1_rich-text">
+  <rich-text  class="{{item.props['class']}}"  style="{{item.props['style']}}"  nodes="{{item.props['nodes']}}"  onTap="{{item.props['onTap']}}"  onTouchStart="{{item.props['onTouchStart']}}"  onTouchMove="{{item.props['onTouchMove']}}"  onTouchEnd="{{item.props['onTouchEnd']}}"  onTouchCancel="{{item.props['onTouchCancel']}}"  onLongTap="{{item.props['onLongTap']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </rich-text>
+</template>
+  
+  <template name="REMAX_TPL_1_scroll-view">
+  <scroll-view  class="{{item.props['class']}}"  style="{{item.props['style']}}"  scroll-x="{{item.props['scroll-x']}}"  scroll-y="{{item.props['scroll-y']}}"  upper-threshold="{{item.props['upper-threshold']}}"  lower-threshold="{{item.props['lower-threshold']}}"  scroll-top="{{item.props['scroll-top']}}"  scroll-left="{{item.props['scroll-left']}}"  scroll-into-view="{{item.props['scroll-into-view']}}"  scroll-with-animation="{{item.props['scroll-with-animation']}}"  scroll-animation-duration="{{item.props['scroll-animation-duration']}}"  enable-back-to-top="{{item.props['enable-back-to-top']}}"  trap-scroll="{{item.props['trap-scroll']}}"  onScrollToUpper="{{item.props['onScrollToUpper']}}"  onScrollToLower="{{item.props['onScrollToLower']}}"  onScroll="{{item.props['onScroll']}}"  onTouchStart="{{item.props['onTouchStart']}}"  onTouchMove="{{item.props['onTouchMove']}}"  onTouchEnd="{{item.props['onTouchEnd']}}"  onTouchCancel="{{item.props['onTouchCancel']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </scroll-view>
+</template>
+  
+  <template name="REMAX_TPL_1_slider">
+  <slider  class="{{item.props['class']}}"  style="{{item.props['style']}}"  name="{{item.props['name']}}"  min="{{item.props['min']}}"  max="{{item.props['max']}}"  step="{{item.props['step']}}"  disabled="{{item.props['disabled']}}"  value="{{item.props['value']}}"  show-value="{{item.props['show-value']}}"  active-color="{{item.props['active-color']}}"  background-color="{{item.props['background-color']}}"  track-size="{{item.props['track-size']}}"  handle-size="{{item.props['handle-size']}}"  handle-color="{{item.props['handle-color']}}"  onChange="{{item.props['onChange']}}"  onChanging="{{item.props['onChanging']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </slider>
+</template>
+  
+  <template name="REMAX_TPL_1_slot-0">
+  <slot-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </slot-0>
+</template>
+  
+  <template name="REMAX_TPL_1_src-0">
+  <src-0 ref="{{item.props['__ref']}}">
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <block a:if="{{item.props['slot']}}">
+        <view  slot="{{item.props['slot']}}"
+           disable-scroll="{{item.props['disable-scroll']}}"
+           hover-class="{{item.props['hover-class']}}"
+           hover-start-time="{{item.props['hover-start-time']}}"
+           hover-stay-time="{{item.props['hover-stay-time']}}"
+           hidden="{{item.props['hidden']}}"
+           class="{{item.props['class']}}"
+           style="{{item.props['style']}}"
+           animation="{{item.props['animation']}}"
+           hover-stop-propagation="{{item.props['hover-stop-propagation']}}"
+           onTap="{{item.props['onTap']}}"
+           onTouchStart="{{item.props['onTouchStart']}}"
+           onTouchMove="{{item.props['onTouchMove']}}"
+           onTouchEnd="{{item.props['onTouchEnd']}}"
+           onTouchCancel="{{item.props['onTouchCancel']}}"
+           onLongTap="{{item.props['onLongTap']}}"
+           onTransitionEnd="{{item.props['onTransitionEnd']}}"
+           onAnimationIteration="{{item.props['onAnimationIteration']}}"
+           onAnimationStart="{{item.props['onAnimationStart']}}"
+           onAnimationEnd="{{item.props['onAnimationEnd']}}"
+           onAppear="{{item.props['onAppear']}}"
+           onDisappear="{{item.props['onDisappear']}}"
+           onFirstAppear="{{item.props['onFirstAppear']}}"
+          >
+          <block a:for="{{item.children}}" key="{{item.id}}">
+            <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+          </block>
+        </view>
+      </block>
+      <block a:else>
+        <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+      </block>
+    </block>
+    
+  </src-0>
+</template>
+  
+  <template name="REMAX_TPL_1_swiper">
+  <swiper  class="{{item.props['class']}}"  style="{{item.props['style']}}"  indicator-dots="{{item.props['indicator-dots']}}"  indicator-color="{{item.props['indicator-color']}}"  indicator-active-color="{{item.props['indicator-active-color']}}"  active-class="{{item.props['active-class']}}"  changing-class="{{item.props['changing-class']}}"  autoplay="{{item.props['autoplay']}}"  current="{{item.props['current']}}"  duration="{{item.props['duration']}}"  interval="{{item.props['interval']}}"  circular="{{item.props['circular']}}"  vertical="{{item.props['vertical']}}"  previous-margin="{{item.props['previous-margin']}}"  next-margin="{{item.props['next-margin']}}"  acceleration="{{item.props['acceleration']}}"  disable-programmatic-animation="{{item.props['disable-programmatic-animation']}}"  onChange="{{item.props['onChange']}}"  onTransition="{{item.props['onTransition']}}"  onAnimationEnd="{{item.props['onAnimationEnd']}}"  disable-touch="{{item.props['disable-touch']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <swiper-item key="{{item.props.key}}">
+        <block a:for="{{item.children}}" key="{{item.id}}">
+          <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+        </block>
+      </swiper-item>
+    </block>
+    
+  </swiper>
+</template>
+  
+  <template name="REMAX_TPL_1_switch">
+  <switch  class="{{item.props['class']}}"  style="{{item.props['style']}}"  name="{{item.props['name']}}"  checked="{{item.props['checked']}}"  disabled="{{item.props['disabled']}}"  color="{{item.props['color']}}"  onChange="{{item.props['onChange']}}"  controlled="{{item.props['controlled']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </switch>
+</template>
+  
+  <template name="REMAX_TPL_1_text">
+  <text  class="{{item.props['class']}}"  style="{{item.props['style']}}"  selectable="{{item.props['selectable']}}"  space="{{item.props['space']}}"  decode="{{item.props['decode']}}"  number-of-lines="{{item.props['number-of-lines']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </text>
+</template>
+  
+  <template name="REMAX_TPL_1_textarea">
+  <textarea  class="{{item.props['class']}}"  style="{{item.props['style']}}"  name="{{item.props['name']}}"  value="{{item.props['value']}}"  placeholder="{{item.props['placeholder']}}"  placeholder-style="{{item.props['placeholder-style']}}"  placeholder-class="{{item.props['placeholder-class']}}"  disabled="{{item.props['disabled']}}"  maxlength="{{item.props['maxlength']}}"  focus="{{item.props['focus']}}"  auto-height="{{item.props['auto-height']}}"  show-count="{{item.props['show-count']}}"  controlled="{{item.props['controlled']}}"  onInput="{{item.props['onInput']}}"  onFocus="{{item.props['onFocus']}}"  onBlur="{{item.props['onBlur']}}"  onConfirm="{{item.props['onConfirm']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </textarea>
+</template>
+  
+  <template name="REMAX_TPL_1_video">
+  <video  src="{{item.props['src']}}"  id="{{item.props['id']}}"  poster="{{item.props['poster']}}"  object-fit="{{item.props['object-fit']}}"  initial-time="{{item.props['initial-time']}}"  duration="{{item.props['duration']}}"  controls="{{item.props['controls']}}"  autoplay="{{item.props['autoplay']}}"  direction="{{item.props['direction']}}"  loop="{{item.props['loop']}}"  muted="{{item.props['muted']}}"  show-fullscreen-btn="{{item.props['show-fullscreen-btn']}}"  show-play-btn="{{item.props['show-play-btn']}}"  show-center-play-btn="{{item.props['show-center-play-btn']}}"  show-mute-btn="{{item.props['show-mute-btn']}}"  enable-progress-gesture="{{item.props['enable-progress-gesture']}}"  mobilenet-hint-type="{{item.props['mobilenet-hint-type']}}"  onPlay="{{item.props['onPlay']}}"  onPause="{{item.props['onPause']}}"  onEnded="{{item.props['onEnded']}}"  onTimeUpdate="{{item.props['onTimeUpdate']}}"  onLoading="{{item.props['onLoading']}}"  onError="{{item.props['onError']}}"  onFullScreenChange="{{item.props['onFullScreenChange']}}"  onTap="{{item.props['onTap']}}"  onUserAction="{{item.props['onUserAction']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </video>
+</template>
+  
+  <template name="REMAX_TPL_1_view">
+  <view  slot="{{item.props['slot']}}"  disable-scroll="{{item.props['disable-scroll']}}"  hover-class="{{item.props['hover-class']}}"  hover-start-time="{{item.props['hover-start-time']}}"  hover-stay-time="{{item.props['hover-stay-time']}}"  hidden="{{item.props['hidden']}}"  class="{{item.props['class']}}"  style="{{item.props['style']}}"  animation="{{item.props['animation']}}"  hover-stop-propagation="{{item.props['hover-stop-propagation']}}"  onTap="{{item.props['onTap']}}"  onTouchStart="{{item.props['onTouchStart']}}"  onTouchMove="{{item.props['onTouchMove']}}"  onTouchEnd="{{item.props['onTouchEnd']}}"  onTouchCancel="{{item.props['onTouchCancel']}}"  onLongTap="{{item.props['onLongTap']}}"  onTransitionEnd="{{item.props['onTransitionEnd']}}"  onAnimationIteration="{{item.props['onAnimationIteration']}}"  onAnimationStart="{{item.props['onAnimationStart']}}"  onAnimationEnd="{{item.props['onAnimationEnd']}}"  onAppear="{{item.props['onAppear']}}"  onDisappear="{{item.props['onDisappear']}}"  onFirstAppear="{{item.props['onFirstAppear']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </view>
+</template>
+  
+  <template name="REMAX_TPL_1_web-view">
+  <web-view  src="{{item.props['src']}}"  onMessage="{{item.props['onMessage']}}"  data-rid="{{item.props['data-rid']}}" >
+    
+    <block a:for="{{item.children}}" key="{{item.id}}">
+      <template is="REMAX_TPL_1_CONTAINER" data="{{item: item}}" />
+    </block>
+    
+  </web-view>
+</template>
+  
+
+<template name="REMAX_TPL_1_plain-text">
+  <block>{{item.text}}</block>
+</template>
+
+<template name="REMAX_TPL_1_CONTAINER">
+  <template is="{{'REMAX_TPL_1_' + item.type}}" data="{{item: item}}" />
+</template>

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.js
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.js
@@ -1,0 +1,55 @@
+import '../npm/remax/esm/render.js';
+import { createElement } from 'react';
+import '../npm/remax/esm/createAppConfig.js';
+import '../npm/remax/esm/Platform.js';
+import '../npm/remax/esm/createHostComponent.js';
+import createPageConfig from '../npm/remax/esm/createPageConfig.js';
+import createNativeComponent from '../npm/remax/esm/createNativeComponent.js';
+import '../npm/remax/esm/index.js';
+import View from '../npm/remax/esm/adapters/alipay/components/View.js';
+import '../npm/remax/esm/adapters/alipay/components/ScrollView.js';
+import '../npm/remax/esm/adapters/alipay/components/Swiper.js';
+import '../npm/remax/esm/adapters/alipay/components/SwiperItem.js';
+import '../npm/remax/esm/adapters/alipay/components/MovableView.js';
+import '../npm/remax/esm/adapters/alipay/components/MovableArea.js';
+import '../npm/remax/esm/adapters/alipay/components/CoverView.js';
+import '../npm/remax/esm/adapters/alipay/components/CoverImage.js';
+import '../npm/remax/esm/adapters/alipay/components/Icon.js';
+import '../npm/remax/esm/adapters/alipay/components/Text.js';
+import '../npm/remax/esm/adapters/alipay/components/RichText.js';
+import '../npm/remax/esm/adapters/alipay/components/Progress.js';
+import '../npm/remax/esm/adapters/alipay/components/Button.js';
+import '../npm/remax/esm/adapters/alipay/components/CheckboxGroup.js';
+import '../npm/remax/esm/adapters/alipay/components/Checkbox.js';
+import '../npm/remax/esm/adapters/alipay/components/Form.js';
+import '../npm/remax/esm/adapters/alipay/components/Input.js';
+import '../npm/remax/esm/adapters/alipay/components/Label.js';
+import '../npm/remax/esm/adapters/alipay/components/Picker.js';
+import '../npm/remax/esm/adapters/alipay/components/PickerView.js';
+import '../npm/remax/esm/adapters/alipay/components/PickerViewColumn.js';
+import '../npm/remax/esm/adapters/alipay/components/RadioGroup.js';
+import '../npm/remax/esm/adapters/alipay/components/Radio.js';
+import '../npm/remax/esm/adapters/alipay/components/Slider.js';
+import '../npm/remax/esm/adapters/alipay/components/Switch.js';
+import '../npm/remax/esm/adapters/alipay/components/TextArea.js';
+import '../npm/remax/esm/adapters/alipay/components/Navigator.js';
+import '../npm/remax/esm/adapters/alipay/components/Image.js';
+import '../npm/remax/esm/adapters/alipay/components/Map.js';
+import '../npm/remax/esm/adapters/alipay/components/Canvas.js';
+import '../npm/remax/esm/adapters/alipay/components/WebView.js';
+import '../npm/remax/esm/adapters/alipay/components/Lifestyle.js';
+import '../npm/remax/esm/adapters/alipay/components/ContactButton.js';
+import '../npm/remax/esm/adapters/alipay/components/Video.js';
+import '../npm/remax/esm/adapters/alipay/api.js';
+import G from '../components/g/index.js';
+
+var B = createNativeComponent('b-0');
+
+
+var _page = function _page() {
+  return createElement(View, null, createElement(G, null), createElement(B, null));
+};
+
+var index4 = Page(createPageConfig(_page));
+
+export default index4;

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.json
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/expected/pages/index4.json
@@ -1,0 +1,8 @@
+{
+  "usingComponents": {
+    "a-0": "../components/a/index",
+    "b-0": "../components/b/index",
+    "c-0": "../components/c/index",
+    "d-0": "../components/d/index"
+  }
+}

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/src/app.config.js
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/src/app.config.js
@@ -1,5 +1,5 @@
 module.exports = {
-  pages: ['pages/index', 'pages/index2', 'pages/index3'],
+  pages: ['pages/index', 'pages/index2', 'pages/index3', 'pages/index4'],
   plugins: {
     myPlugin: {
       version: '0.8.1',

--- a/packages/remax-cli/src/__tests__/fixtures/nativeComponent/src/pages/index4.tsx
+++ b/packages/remax-cli/src/__tests__/fixtures/nativeComponent/src/pages/index4.tsx
@@ -1,0 +1,11 @@
+import * as React from 'react';
+import { View } from 'remax/alipay';
+import G from '@/components/g';
+import B from '@/components/b';
+
+export default () => (
+  <View>
+    <G />
+    <B />
+  </View>
+);

--- a/packages/remax-cli/src/build/plugins/nativeComponents/index.ts
+++ b/packages/remax-cli/src/build/plugins/nativeComponents/index.ts
@@ -100,14 +100,20 @@ export default (
     },
     generateBundle(_, bundle) {
       const importers = getImporters();
-      const collected: Set<string> = new Set();
+      const collected: Map<string, Set<string>> = new Map();
 
       const collectPages = (page: string, importer: string) => {
-        if (collected.has(importer)) {
-          return;
-        }
+        const pageCollected = collected.get(page);
 
-        collected.add(importer);
+        if (pageCollected) {
+          if (pageCollected.has(importer)) {
+            return;
+          }
+
+          pageCollected.add(importer);
+        } else {
+          collected.set(page, new Set([importer]));
+        }
 
         const nativeImporter = importers.get(
           searchFile(

